src/vaadin-grid.html

```diff
@@ -427,6 +427,7 @@
         super.attributeChangedCallback(name, oldValue, newValue);
         if (name === 'dir') {
           this.__isRTL = newValue === 'rtl';
+          this._updateScrollerMeasurements && this._updateScrollerMeasurements();
```

Comment:
Why the check for `_updateScrollerMeasurements`? Is there a chance it doesn't exist (it's defined in scroll-mixin)?

Reply:
Discussed with Serhii as well. Removed now.